sync/syncer.go

```diff
@@ -78,6 +78,14 @@ var (
 	errTooManyAtxs = errors.New("too many atxs in blocks")
 )
 
+type Status int
+
+const (
+	Pending    Status = 0
+	InProgress Status = 1
+	Done       Status = 2
```

Comment:
Consider changing names.
Maybe GossipStatues is {on / open, off / closed, in progress / opening / on-off quantum state}